db/data_migration/20160311093609_generate_content_ids_on_attachments.rb

```diff
@@ -0,0 +1,6 @@
+require 'securerandom'
+
+Attachment.where(content_id: nil).find_each do |attachment|
```

Comment:
We should carry on filling the field with Ruby UUIDs. There's no really good reason not to do this. There's enough doubt about the alternative for for this to be the correct way.